### DISCUSSION TITLE
`Internal Server Error` after updating | v1.33.1 -> v1.36.1

### DISCUSSION THREAD

**Discussion by mateusfg7** [umami-software/umami#1377] [Q&A] [answered]
I have an umami instance on Heroku, it was in [v1.33.1](https://github.com/umami-software/umami/releases/tag/v1.33.1), today I update to [v1.36.1](https://github.com/umami-software/umami/releases/tag/v1.36.1), after the deployment, the website appears the error `Internal Server Error`

![image](https://user-images.githubusercontent.com/40613276/182244290-6c4850a7-eabb-4ccf-bac5-cea356001af8.png)

Anyone knows how to solve this problem?

**Comment by mikecao** [comment, marked as answer]
Heroku does not work with recent versions of Next.js, which is what the latest version of Umami uses.

**Comment by H0rn0chse** [comment]
The Internal Server Error should be resolved by https://github.com/umami-software/umami/pull/1380
